src/Symfony/Component/Messenger/DataCollector/MessengerDataCollector.php

```diff
@@ -117,13 +117,15 @@ public function getExceptionsCount(string $bus = null): int
         return $count;
     }
 
-    public function getMessages(string $bus = null): iterable
+    public function getMessages(string $bus = null): array
```

Comment:
getMessages() in 4.1

https://github.com/symfony/symfony/blob/913781baac1137dbf0c175298d711560f9ab1cb7/src/Symfony/Component/Messenger/DataCollector/MessengerDataCollector.php#L129

Reply:
why change? the BC break is allowed and the new interface better, don't you think?

Reply:
OH, in order to "count" on the result. OK.